sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLRunDAO.java

```diff
@@ -502,6 +502,9 @@ public long save(Run run) throws IOException {
               l.setId(sequencerPartitionContainerDAO.save(l));
             }
             SEQ_PART_CONTAINER_WRITER.saveAll(template, run.getId(), run.getSequencerPartitionContainers());
+            for (SequencerPartitionContainer<SequencerPoolPartition> container : run.getSequencerPartitionContainers()) {
+              sequencerPartitionContainerDAO.save(container);
```

Comment:
These are already saved 4 lines higher up

Was the problem with saving the containers in a run, or with saving the pools within lanes?